gpu-enabler/src/main/scala/com/ibm/gpuenabler/GPUMemoryManager.scala

```diff
@@ -121,20 +158,56 @@ private[gpuenabler] class GPUMemoryManager(val executorId : String,
       }
     }
   }
+  def autoUnCacheGPU(rddId : Int): Unit = {
+    autoCachedGPURDDs.foreach(s =>
+      if(cachedGPURDDs.contains(s)) {
+          autoCachedGPURDDs -= s
+      }
+    )
+    if(autoCachedGPURDDs.size > 0){
+     val rddId = autoCachedGPURDDs.head
```

Comment:
nit: Can we format this block with better indent?

Reply:
Yes you can. 
This indent has no intention.